pkg/resourceinterpreter/defaultinterpreter/default.go

```diff
@@ -123,3 +126,14 @@ func (e *DefaultInterpreter) ReflectStatus(object *unstructured.Unstructured) (s
 	// for resource types that don't have a build-in handler, try to collect the whole status from '.status' filed.
 	return reflectWholeStatus(object)
 }
+
+// InterpretHealthy returns the healthy check result of the object.
+func (e *DefaultInterpreter) InterpretHealthy(object *unstructured.Unstructured) (bool, error) {
+	handler, exist := e.healthyHandlers[object.GroupVersionKind()]
+	if exist {
+		return handler(object)
+	}
+
+	// for resource types that don't need to check healthy, return true directly
```

Comment:
We can't assume the resource's default to `healthy`.
For resource types that we don't know, we should return error.

Reply:
Thanks for pointing out, it has been modified in my latest commit.